Sources/Common/Background Queue/ApiSyncQueueRunner.swift

```diff
@@ -7,7 +7,7 @@ open class ApiSyncQueueRunner {
     public let logger: Logger
     private let httpClient: HttpClient
 
-    public let failureIfDontDecodeTaskData: Result<Void, CustomerIOError> = .failure(.http(.noRequestMade(nil)))
+    public let failureIfDontDecodeTaskData: Result<Void, HttpRequestError> = .failure(.noRequestMade(nil))
```

Comment:
since the public signature changed is this going to be a breaking change?

Reply:
Most of the code in the `Common` module are `public` because that's the only way that all of the other modules in the project are able to access the code. But, we do not document any of the code in the `Common` module. Therefore, we *hope* that customers do not use any of the public classes/functions/properties in this module. 

So, no. If you search for `APITest` files in this project, those are the `public` classes/functions/properties that we are tracking. Any modifications to any of those will result in a breaking change.